src/components/PostCard/PostCard.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  PostCardWrapper,
+  Image,
+  DateCategoryWrapper,
+  Date,
+  Title,
+  IconWrapper,
+  MoreLink,
+  MoreLinkWrapper
+} from './PostCard.styles';
+import { ArrowCircleRight, Share, Bookmark } from '../Icons';
+
+const PostCard = ({
+  coverUrl,
+  category,
+  date,
+  title,
+  postLink,
+  shareLink,
+  bookmark
+}) => {
+  return (
+    <PostCardWrapper>
+      <Image src={coverUrl} alt={title} />
+      <DateCategoryWrapper>
+        <p>{category}</p>
+        <Date>{date}</Date>
+      </DateCategoryWrapper>
+      <Title>{title}</Title>
+      <IconWrapper>
+        <MoreLinkWrapper>
+          <ArrowCircleRight />
+          <MoreLink to={postLink}>Read more</MoreLink>
+        </MoreLinkWrapper>
+        <MoreLinkWrapper>
+          <a href={shareLink} title="Share link">
+            <Share />
+          </a>
+          <a href={bookmark} title="Bookmark link">
```

Comment:
Bookmark and share funcionality. Need to verify how we handle it. For my current knowledge bookmark is a `<a>` tag but sharing the link , should be a `<button>` tag whitch  copy the URL to clipboard or  redirect us to specific webiste.
Everything else is fine.

Please look at [https://www.npmjs.com/package/react-share](url)
and ...[https://stackoverflow.com/questions/54759623/how-to-create-a-bookmark-this-page-button](url) do we want page to be bookmarked or is it a link to list of Bookmarks?!

Reply:
Thank you for the review @admtomas. Right now I will just merge it to develop branch, as I need this component for homepage.  Once I check with Rob about the query, l will work on it again.